micromanager_gui/multid_widget.py

```diff
@@ -98,8 +99,8 @@ def __init__(self, mmcore: RemoteMMCore, parent=None):
         super().__init__(parent)
         self.setup_ui()
 
-        self.pause_Button.released.connect(self._mmc.toggle_pause)
-        self.cancel_Button.released.connect(self._mmc.cancel)
+        self.pause_Button.released.connect(lambda: self._mmc.mda.toggle_pause())
```

Comment:
i'm curious, did you find that the lambda was necessary here for this to work?  I've definitely seen that before.  Always kinda wonder what the circumstances are though